composer.json

```diff
@@ -14,7 +14,7 @@
         "symfony/stopwatch": "^4.0|^5.0"
     },
     "require-dev": {
-        "atoum/atoum": "^3.1",
+        "atoum/atoum": "^4.0",
```

Comment:
Required in order to be able to run tests on PHP 8.0
The downside is that it's not anymore able to run on PHP 7.1 with this version, but PHP 7.1 is not tested via GA.

Reply:
This change would imply a change of the PHP minimal required version in composer.json, from '>=7.1' to `>=7.2` which is a BC

Reply:
That's a dev package ;)